### DISCUSSION TITLE
New attributes to value:block tag for curators

### DISCUSSION THREAD

**Discussion by thebells1111** [Podcastindex-org/podcast-namespace#499] [Enhancement Proposal]
As we start work on remoteItem, there will be people who have a particular knack for curation of lists and recommendations that others will want to reward. The prospect of reward will also encourage participation in these new tags. As it stands, remoteItem doesn't allow for the value block, and it shouldn't, because curating a list doesn't warrant taking a share of the podcasters or musicians value. However, a listener may be compelled to want to provide value to the curator. I'm trying to think of a way to have a remoteItem feed with a value block in the channel to allow contributions to the curator. Perhaps a new attribute in the value block, something like "tip" or "curator" in addition to "split" or "fee". Or maybe even better would be a standard or protocol for the apps that if a feed has remoteItems, the value block in the channel or remoteItem are never to be inserted into the splits, and the splits for boosts and streaming is derived exclusively from the remoteItem that is fetched based on the guid.  However, the value block in the remoteItem feed can be surfaced in an app as perhaps something to "Tip" the curator, so the choice to send value to the curator is a conscience one, and isn't taking any of the value that would normally go to the podcaster's value block.

**Comment by adamc199** [comment]
I feel it is critical to find a way for curators to participate in the value flow. I do not agree with calling anything a 'tip'

Curators will be extremely valuable to music creators. Good curation is hard to do and valuable.

**Comment by ericpp** [comment]
I like that idea. This would let the podcaster reserve a split for curators and used when their podcast is played from a curated feed?

**Reply by thebells1111** [reply]
That's one way, of doing. The podcaster can have a split that has some way that this goes to whoever the curator is, and if that split isn't present, then the curator doesn't get a cut. I do think the podcaster should get a say in whether or not a curator can dip into their value block.
